.gitignore

```diff
@@ -383,6 +383,13 @@ experimental/generator-dotnet-yeoman/node_modules
 
 # Allow the root-level /packages/ directory or it gets confused with NuGet directories
 !/[Pp]ackages/*
+!/tests/[Pp]ackages/*
+
+# Allow the packages directory under tests
```

Comment:
There are a couple duplicates of the same directory, maybe can just use !/tests/**/[Pp]ackages/*?

Reply:
Removed duplicates. There is one instance of !/tests/unit/[Pp]ackages/*. Did you want to remove all references to tests/unit?

Reply:
No we should keep the one! 

Are there any instances of `!/tests/[Pp]ackages/*` or can that be removed?

Reply:
The Telephony tests are there. I will move them out after this merge and once a few PRs in flight settle down. Too many conflicts otherwise :( 

Will make sure to remove this line with that change.